.github/workflows/nightly.yml

```diff
@@ -1,5 +1,7 @@
 name: Nightly Builds
 
+permissions: {}
```

Comment:
Isn't this just syntactically a no-op, because not specifying the element is the same as setting it as empty. According to https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#defining-access-for-the-github_token-scopes:

> all of those that are not specified are set to `none`.

so not sure, if should or will change anything?

Reply:
If you don't set this, all permissions are available. As an example, we don't set permissions in publish.yml (before this PR set them) and that job reads secrets.

Reply:
I think you're right. I tried it out in one of my projects and the permissions did not seem to affect secrets access.